core/tauri/src/scope/shell.rs

```diff
@@ -58,7 +58,7 @@ impl From<Vec<String>> for ExecuteArgs {
 }
 
 /// Shell scope configuration.
-#[derive(Debug, Clone)]
+#[derive(Debug, Default, Clone)]
```

Comment:
This derive will have the `open` field set to `None` by default, which means that it is _**not**_ matched through the default regex. It should probably match the default of `^((mailto:\w+)|(tel:\w+)|(https?://\w+)).+`. We may want to adjust how we create it from the config so that we don't need to have the "default" filtering regex in more than 1 place, to reduce future duplication mistakes.